### ISSUE TITLE
Any plan to release sparkleshare-mac-3.38.zip?

### ISSUE THREAD

**Issue by Thireus** [hbons/SparkleShare#1960]
Thanks a lot for your work @hbons! Do you plan to release the compiled version of SparkleShare 3.38 for macOS?

**Comment by uche-eis** [comment]
Indeed, I'm not able to build the Sparkleshare release for Mac (Catalina 10.15.7) . I have XCode, I installed Visual Studio Code, followed the complicated instructions and debugged a variety of security and other roadblocks and I still end up with a Mac app that won't launch (SparkleShare/Mac/bin/ReleaseMac/SparkleShare.app, launched from Finder fails with "You can’t open the application “SparkleShare” because it may be damaged or incomplete.").

**Comment by markusstoll** [comment]
Some goes for me - I am not able to get it running for myself. See  #1932

**Comment by markusstoll** [comment]
see #1964 to make it run under macOS again
